Arrangement_on_surface_2/include/CGAL/Arr_counting_traits_2.h

```diff
@@ -727,18 +667,18 @@ class Arr_counting_traits_2 : public Base_traits {
     { ++m_counter3; return m_object(xcv1, ce1, xcv2, ce2); }
   };
 
-  /*! A functor that compares the x-coordinates of curve ends near the
+  /*! A functor that compares the \f$x\f$-coordinates of curve ends near the
```

Comment:
here the `x` is shown as a formula but at other places the `y` of `y-coordinate` is not shown as a formula ...